scripts/prepare.js

```diff
@@ -0,0 +1,6 @@
+// prepare.js
+try {
```

Comment:
Unless really needed, I would prefer to avoid adding an additional folder for scripts, it might be a door for more custom scripts and non-trivial installation. Not critical, just checking whether this can be solved locally 👇

Can't we do something like:

"prepare" : "isCI || husky install"

?

cc @idobetesh @rluvaton

Reply:
no, because it happens on local machines as well